src/AppContainer.prod.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+
+'use strict';
+
+const React = require('react');
+const { Component } = React;
+
+class AppContainer extends Component {
+  render() {
+    if (this.props.component) {
+      return <this.props.component {...this.props.prop} />;
```

Comment:
I'll switch over to use children as a primary workaround for the bug and I think if the component/props way gets dropped that's just better. Assuming it has no other purpose.